privacyidea/static/css/signin.css

```diff
@@ -126,3 +126,8 @@ select.ng-invalid:focus:not(#username):not(#password){
   -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075), 0 0 6px #ce8483;
           box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075), 0 0 6px #ce8483;
 }
+
+.pidropdown {
+    padding-top: 7px;
+    padding-bottom: 7px;
+}
```

Comment:
We could also add
`.navbar-nav .btn-group {
    padding-top: 7px;
}`
to avoid adding an extra class.

Reply:
@Mipronimo what do you think?

Reply:
And since my comment on #1581 didn't appear: isn't the `signin.css` for the login page? Shouldn't we use a generic bootstrap override?

Reply:
I do not think we should overcomplicate things ;-)

Reply:
I will put the navbar css in an extra css file ``menu.css``.
We already have a few own css files. This way it might make more sense, that users can more easily adopt the css (customization) without the need to maintain the complete file.

Reply:
@plettich I added your comments. Please review and merge in branch-3.0
By introducing the menu.css we also get rid of merge conflicts. And it makes customization easier.